wiki/Main_Page/en.md

```diff
@@ -63,11 +62,12 @@ Sections: ([Compose](/wiki/Beatmap_Editor/Compose) • [Design](/wiki/Beatmap_Ed
 
 # How You Can Help
 
-<p>
+<!-- This p tag is required. -->
+<p class="osu-md__paragraph">
 If you're interested in shaping the future of osu!, you can help by contributing to any of its projects that interest you.
```

Comment:
reads poorly, sort of implies that these are the only ways to contribute to osu, and lists an odd selection of projects too (most people contributing back to the game aren't involved with any of these...)

Reply:
Alright, I'll remove the section instead.